sonic_y_cable/y_cable.py

```diff
@@ -92,9 +92,9 @@
 
 MAX_NUM_LANES = 4
 
-# definitions for enable/disable mode for autoswitch feature
-ENABLE_AUTO_SWITCH = 1
-DISABLE_AUTO_SWITCH = 0
+# definitions for switching modes inside muxcable
+SWITCHING_MODE_AUTO = 1
+SWITCHING_MODE_MANUAL = 0
```

Comment:
Suggest arranging these in numerical order

Reply:
fixed